src/components/Textarea/Textarea.js

```diff
@@ -1,14 +1,40 @@
 // components/Task.js
 import * as React from 'react';
-import { SafeAreaView, View, StyleSheet,TextInput } from 'react-native';
+import {SafeAreaView, View, StyleSheet, TextInput} from 'react-native';
 // import { styles } from '../constants/globalStyles';
 
-export default function Example({phrase,editable,onChange = () => null, placeholder}) {
-
+export default function Example({
+  phrase,
+  editable,
+  onChange = () => null,
+  placeholder,
+  backgroundColor,
+  textColor,
```

Comment:
maybe just use default value `'#111827'`
`textColor='textColor,'`
and you can get rid of 
`color: textColor ? textColor : '#111827'`
and just use
`color: textColor`